src/frontend/staff-portal/src/assets/config/oidc.json

```diff
@@ -0,0 +1,12 @@
+{
+    "postLoginRoute": "/ticket",
+    "authority": "https://dev.oidc.gov.bc.ca/auth/realms/ezb8kej4",
+    "redirectUrl": "http://localhost:4200",
+    "postLogoutRedirectUri": "http://localhost:4200",
+    "autoUserInfo": false,
+    "clientId": "tco-staff-portal",
+    "scope": "openid profile",
+    "responseType": "code",
+    "prompt": "consent",
+    "kc_idp_hint": "bceid-basic"
```

Comment:
This is wrong. We dont use bceid, we use idir. This should be `"kc_idp_hint": "idir"`